### PR TITLE
Update and move Medium challenge to early challenges, improve first challenge copy

### PR DESCRIPTION
**Pull request by QuincyLarson**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
#### Description

Update Medium challenge, move it to early challenges, improve first challenge copy

### CODE REVIEW

File: seed/challenges/00-getting-started/getting-started.json [hunk 1]
```diff
@@ -21,15 +21,15 @@
           ""
         ],
         [
-          "//i.imgur.com/D7Y5luw.jpg",
-          "A graph of the rate of job growth against growth in computer science degree graduates. There are 1.4 million jobs and only 400 thousand people to fill them.",
-          "There are thousands of coding jobs currently going unfilled, and the demand for coders grows every year.",
+          "//i.imgur.com/WD3STY6.jpg",
+          "Photos of three campers who've gotten jobs after learning to code at Free Code Camp.",
```
Comment (raisedadead) [inline seed/challenges/00-getting-started/getting-started.json]:
```diff
-gotten
+got
```

Neutralizing language for non-native english speakers?

File: seed/challenges/00-getting-started/getting-started.json [hunk 2]
```diff
@@ -21,15 +21,15 @@
...
+          "Free Code Camp is a proven way to become a professional software developer. Thousands of people like you have gotten developer jobs after learning with Free Code Camp.",
```
Comment (raisedadead) [inline seed/challenges/00-getting-started/getting-started.json]:
```diff
-gotten
+got
```

Neutralizing for non-native english speakers?

File: seed/challenges/00-getting-started/getting-started.json (hunk 1 above)
Comment (alayek) [inline seed/challenges/00-getting-started/getting-started.json]: I was thinking let's not even get into that debate. We could simply write "who've received job offer"

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @QuincyLarson.

**Pull request merged**